### PR TITLE
[blog] update Unlocking the Power of Hybrid Search 

### PR DESCRIPTION
**Pull request by sebawita**
### What's being changed:

Update content

### Type of change:

- [x] **Content** updates – **blog**, **podcast** (non-breaking change to add/update content)

### How Has This Been Tested?

- [x] **Github action** – automated build completed without errors
- [x] **Local build** - the site works as expected when running `yarn start`

### CODE REVIEW

File: blog/2023-08-29-hybrid-search-fusion/index.mdx [hunk 1]
```diff
@@ -223,23 +223,121 @@ The `relativeScoreFusion` algorithm was added in Weaviate version `1.20`.
...
+    <th>Search Type</th>
+    <th>(id): score</th><th>(id): score</th><th>(id): score</th><th>(id): score</th><th>(id): score</th>
+  </tr>
+  <tr>
+    <td>Keyword</td>
+    <td>(1): 0.0164</td><td>(0): 0.0161</td><td>(2): 0.0159</td><td>(4): 0.0156</td><td>(3): 0.0154</td>
+  </tr>
+  <tr>
+    <td>Vector</td>
+    <td>(1): 0.0164</td><td>(0): 0.0161</td><td>(2): 0.0159</td><td>(4): 0.0156</td><td>(3): 0.0154</td>
```
Comment (dirkkul) [inline blog/2023-08-29-hybrid-search-fusion/index.mdx:283]: docids are the same for vector and keyword, copy&paste error?

File: blog/2023-08-29-hybrid-search-fusion/index.mdx [hunk 2]
```diff
@@ -223,23 +223,121 @@ The `relativeScoreFusion` algorithm was added in Weaviate version `1.20`.
 
 In contrast  to `rankedFusion`, however, `relativeScoreFusion` derives each objects score by *normalizing* the metrics output by the vector search and keyword search respectively. The highest value becomes 1, the lowest value becomes 0, and others end up in between according to this scale. The total score is thus calculated by a scaled sum of normalized vector similarity and normalized BM25 score.
 
+## Full example
+
+After reviewing each algorithm, let’s go through a complete example to demonstrate the difference between them.
+
+Let's say that a search returns **five objects** with **document id** (from 0 to 5), and **scores** from **keyword** and **vector search**, **ordered by score**:
```
Comment (dirkkul) [inline blog/2023-08-29-hybrid-search-fusion/index.mdx:230]: from 0 to 4

File: blog/2023-08-29-hybrid-search-fusion/index.mdx [hunk 3]
```diff
@@ -223,23 +223,121 @@ The `relativeScoreFusion` algorithm was added in Weaviate version `1.20`.
...
+  </tr>
+  <tr>
+    <td>Relative</td>
+    <td>(1): 1.986</td><td>(0): 1.504</td><td>(2): 1.450</td><td>(4): 1.018</td><td>(3): 0.0</td>
+  </tr>
+</table>
+
+### What can we learn from this?
+
+For the vector scores, the first 4 results (IDs 2, 4, 0, 1) were almost identical, and all of them were good results, while for the keyword search, the first result (ID 1) was much better than the rest. This is captured much better in the final result of relative_score_fusion, where Document 1 is the top score. This is justified because this document was the best result in the keyword search with a big gap to the next-best score and in the top group of vector search. In contrast, for ranked fusion, Document 2 is the top result, closely followed by 1 and 0.
```
Comment (dirkkul) [inline blog/2023-08-29-hybrid-search-fusion/index.mdx:331]: we sometimes use ID 1 and sometimes document 1, should we unify?

Comment (dirkkul) [inline blog/2023-08-29-hybrid-search-fusion/index.mdx:331]: replace relative_score_fusion with `relativeScoreFusion`?

File: blog/2023-08-29-hybrid-search-fusion/index.mdx [hunk 4]
```diff
@@ -223,23 +223,121 @@ The `relativeScoreFusion` algorithm was added in Weaviate version `1.20`.
```
Comment (dirkkul) [inline blog/2023-08-29-hybrid-search-fusion/index.mdx:223]: I'd move the picture a bit up, maybe directly to Fusion algo?

**Review by sebawita — COMMENTED** [review]
I've updated all parts you highlighted 👍

**Pull request merged**